### PR TITLE
Table data full viewer fix

### PR DESCRIPTION
**Pull request by will-moore**
# What this PR does

Fixes a bug reported in
https://www.openmicroscopy.org/community/viewtopic.php?f=4&t=8345
where the Tables data is not displayed in the full image viewer.

# Testing this PR

1. Create tables data on Sreen AND on a Plate, with rows corresponding to Wells, as described in http://help.openmicroscopy.org/scripts.html#metadata
1. Confirm that this data is shown in webclient right panel under Tables tab when Well is selected
1. Open images from that Well in full image viewer and check that Tables data is shown under "Image Information" popup

screen shot 2017-08-16 at 23 08 38

TODO:
 - [ ] Add robot test

### CODE REVIEW

**Comment by eleanorwilliams** [comment]
I tried to test this on http://web-dev-merge.openmicroscopy.org/webclient/ logged in at first as user-4 and then as user-1, but although I could add a csv file to a plate or screen, the populate metadata script failed to create a bulk annotations file and gives an error   `Unable to create table: %s" % name)
omero.util.populate_metadata.MetadataError: Unable to create table: bulk_annotations`. 

I couldn't find any plates with tables already created.

**Comment by will-moore** [comment]
Even Plates with ```bulk_annotations``` attached don't show any Tables data for their Wells. E.g. http://web-dev-merge.openmicroscopy.org/webclient/?show=plate-2072 (user-3)
Not sure what's going on here: Locally I also can't create tables but display of existing tables data is OK.

**Comment by will-moore** [comment]
Table data is now being displayed ok at http://web-dev-merge.openmicroscopy.org/webclient/?show=well-336745 (user-3) and in Full Viewer: http://web-dev-merge.openmicroscopy.org/webclient/img_detail/51637/

**Comment by jburel** [comment]
Table data displayed
The display could be improved in the viewer but not worth spending time on it.
To come robot tests

File: components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/includes/toolbar.html [hunk 1]
```diff
@@ -205,7 +205,7 @@
             {% endif %}
 
             {% if not share %}
-            {% if image and not omeTiffDisabled %}
+            {% if image %}
```
Comment (jburel) [inline components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/includes/toolbar.html]: This commit is not related to the description of the PR
Why did you include it?

Comment (will-moore) [reply components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/includes/toolbar.html]: It seemed so minor as to not justify a separate PR, but I can put it in a separate PR?

**Comment by will-moore** [comment]
Robot test is failing because previous test is leaving the Image Viewer window open, not the webclient.
https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-robotframework/691/robot/Web%20&%20Web/Web_1/View%20Image/Test%20Bulk%20Annotations/
Will fix...

File: components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/includes/toolbar.html (hunk 1 above)
Comment (jburel) [reply components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/includes/toolbar.html]: in the "export" PR discussed this am

**Comment by will-moore** [comment]
Passing now: https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-robotframework/692/robot/Web%20&%20Web/Web_1/View%20Image/Test%20Bulk%20Annotations/

**Comment by mtbc** [comment]
https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-robotframework/693/robot/Web%20&%20Web/Web/View%20Image/Test%20Bulk%20Annotations/ is still good.

**Review by mtbc — APPROVED** [review]

File: components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/includes/toolbar.html (hunk 1 above)
Comment (jburel) [reply components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/includes/toolbar.html]: Could you put the commit in another PR since the change is actually not tested the same way

**Comment by jburel** [comment]
Thanks for rolling back the commit
